src/api/app/models/bs_request.rb

```diff
@@ -636,7 +636,7 @@ def assignreview(opts = {})
   end
 
   def approval_handling(new_approver, opts)
-    raise InvalidStateError, 'request is not in review state' unless state == :review
+    raise InvalidStateError, 'request is not in review state' unless state == :review || state == :new
```

Comment:
`raise InvalidStateError, 'request is not in review state' unless [:review, :new].include?(state)`